transformer_engine/common/include/transformer_engine/permutation.h

```diff
@@ -0,0 +1,19 @@
+/*************************************************************************
+ * Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See LICENSE for license information.
+ ************************************************************************/
+
+#ifndef TRANSFORMER_ENGINE_PERMUTATION_H_
+#define TRANSFORMER_ENGINE_PERMUTATION_H_
+
+#include "transformer_engine.h"
+
+template <typename T, bool FWD>
+void moe_permutation_launcher(const void *input, void *output, const int *sorted_row_id,
```

Comment:
```suggestion
void nvte_permutation(const void *input, void *output, const int *sorted_row_id,
```

Reply:
Hi, all exposed C++ APIs should start with `nvte_`.

Reply:
All current `nvte_` functions have `cuStream_t` as the last arg. Let's keep it consistent.

Reply:
Solved.